test/integration/framework/src/siftool/test_utils.py

```diff
@@ -286,6 +286,9 @@ def __init__(self, cmd, w3_conn: web3.Web3, ctx_eth, abi_provider, operator, sif
         self.generic_erc20_contract = generic_erc20_contract
         self.available_test_eth_accounts = None
 
+    def get_current_block_number(self):
```

Comment:
I assume this returns a number but I need a type hint to verify. Is it a number, bignumber, string?